src/ui-utils/index.ts

```diff
@@ -38,3 +38,14 @@ export function printableBalance(balance: readonly ICoin[]): string {
   if (balance.length === 0) return "–";
   return balance.map(printableCoin).join(", ");
 }
+
+export function parseMsgContract(msg: Uint8Array) {
+  if (!msg) return {};
```

Comment:
A Uint8Array instance is always truthy (also empty ones), so this check is obsolete and should be removed.

```
> !!new Uint8Array()
true
```